File: Examples/DotNet/CSharp/Rows/UngroupRowsInWorksheet.cs
using Aspose.Cloud;
using System;
namespace Aspose.Cells.Cloud.Examples.Rows
{
    class UngroupRowsInWorksheet
    {
        static void Main()
        {
            string dataDir = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            string input = "sample1.xlsx";
            string output = "output.xlsx";
                                    
            Common.StorageService.File.UploadFile(dataDir+input, input, storage: Common.STORAGE);

            string sheetName = "Sheet1";

            Common.CellsService.WorksheetColumns.UngroupWorksheetRows(input, sheetName, 1, 4, true, Common.FOLDER, storage: Common.STORAGE);

            Common.StorageService.File.DownloadFile(input, dataDir + output, storage: Common.STORAGE);

        }
    }
}

File: Examples/Android/app/src/main/java/com/aspose/cells/cloud/examples/rows/UngroupRowsInWorksheet.java
package com.aspose.cells.cloud.examples.rows;


import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.io.File;
import com.aspose.cells.cloud.examples.R;
import android.content.Context;


public class UngroupRowsInWorksheet {

	public static void execute(Context context) throws IOException {
		//ExStart: ungroup-rows-in-excel-worksheet
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			String input = "sample1.xlsx";
			String output = "sample2.xlsx";

			File inputFile = Utils.stream2file("sample1","xlsx", context.getResources().openRawResource(R.raw.sample1));
			File outputFile = Utils.stream2file("sample2","xlsx", context.getResources().openRawResource(R.raw.sample2));


			String sheetName = "Sheet1";
			Integer firstIndex = 1;
			Integer lastIndex = 2;
			Boolean hide = false;

			storageApi.PutCreate(input, null, Utils.STORAGE, inputFile);

			cellsApi.PostUngroupWorksheetRows(input, sheetName, firstIndex, lastIndex, hide, Utils.STORAGE,
					Utils.FOLDER);

			com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload(input, null, Utils.STORAGE);

			Utils.copyInputStreamToFile(sr.getInputStream(),outputFile);
		}

		catch (Exception e) {
			e.printStackTrace();
		}
		
		//ExEnd: ungroup-rows-in-excel-worksheet

	}
}